2/README.md

```diff
@@ -0,0 +1,122 @@
+
+Principles:
+
+* We aim for rough consensus and running code.
+* Specifications are small pieces, made by small teams.
+* Specifications should have a clearly responsible editor.
+* The process should be visible, objective, and accessible to anyone.
+* The process should clearly separate experiments from solutions.
+* The process should allow deprecation of old specifications.
+
+Specifications should take minutes to explain, hours to design, days to write, weeks to prove, months to become mature, and years to replace.
+
+Specifications have no special status except that accorded by the community.
+
+## Architecture
+COSS is designed around fast, easy to use communications tools. Primarily, COSS uses a wiki model for editing and publishing specifications texts.
+
+* The *domain* is the conservancy for a set of specifications in a certain area.
+* Each domain is implemented as an Internet domain, hosting a wiki and optionally other communications tools.
+* Each specification is a set of wiki pages, together with comments, attached files, and other resources.
+* Important specifications may also exist as subdomains, i.e. child wikis.
+
+Individuals can become members of the domain by completing the necessary legal clearance. The copyright, patent, and trademark policies of the domain must be clarified in an Intellectual Property policy that applies to the domain.
+
+Specifications exist as multiple pages, one page per version of the specification (see "Branching and Merging", below), which may be assigned URIs that include an incremental number. Thus, we refer to a specification by specifying its domain, number, and short name. New versions of the same specification will have new numbers. The syntax for a specification reference is:
+
+
+  <domain>/spec:<number>/<shortname>
+
+
+For example, this specification is **rfc.unprotocols.org/spec:2/COSS**. The short form **2/COSS** may be used when referring to the specification from other specifications in the same domain.
+
+Every specification (including branches) carries a different number. Lower numbers indicate more mature specifications, higher numbers indicate more experimental specifications.
+
+## COSS Lifecycle
+Every specification has an independent lifecycle that documents clearly its current status.
+
+A specification has six possible states that reflect its maturity and contractual weight:
+
+TBD: Add lifecycle
+
+### Raw Specifications
+All new specifications are **raw** specifications. Changes to raw specifications can be unilateral and arbitrary. Those seeking to implement a raw specification should ask for it to be made a draft specification. Raw specifications have no contractual weight.
+
+### Draft Specifications
+When raw specifications can be demonstrated, they become **draft** specifications. Changes to draft specifications should be done in consultation with users. Draft specifications are contracts between the editors and implementers.
```

Comment:
I don't understand how one would know when a raw specification "can be demonstrated." Demonstrated? Do they mean *implemented*?

Reply:
I would say it's about an implementation. The next in line would be `stable` which is about the use by others, so it has to work.